src/screens/App/Suggestion/index.tsx

```diff
@@ -1,144 +1,320 @@
 import React from 'react';
-import { Select } from '@components/Select';
+
 import {
   InputContainer,
-  Input,
   Name,
   SuggestionWrapper,
   Container,
-  SelectContainer
+  FormContainer
 } from './styles';
 
 import { DismissKeyboard } from '@utils/DismissKeyBoard';
 import { RoundedButton } from '@components/Button/RoundedButton';
 
+import { useForm } from 'react-hook-form';
+import {
+  SuggestionData,
+  SuggestionSchema
+} from '@screens/App/Suggestion/schema';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { InputForm } from '@components/Input/InputForm';
+import { SelectForm } from '@components/Select/SelectForm';
+import { useToast } from 'native-base';
+import { theme } from '@theme';
+
 export function Suggestion() {
+  const toast = useToast();
+
+  const {
+    control,
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors }
+  } = useForm<SuggestionData>({
+    resolver: zodResolver(SuggestionSchema)
+  });
+
+  function toastSuccess() {
```

Comment:
Mudar função para handleSubmitSuggestion, pq futuramente esse dado vai ser enviado para api

```suggestion
  function handleSubmitSuggestion() {
```